apps/src/componentLibrary/dropdown/actionDropdown/ActionDropdown.tsx

```diff
@@ -0,0 +1,150 @@
+import classNames from 'classnames';
+import React, {useCallback, memo, AriaAttributes} from 'react';
+
+import {ButtonProps} from '@cdo/apps/componentLibrary/button';
+import {
+  DropdownProviderWrapper,
+  useDropdownContext,
+} from '@cdo/apps/componentLibrary/common/contexts/DropdownContext';
+import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
+import CustomDropdown from '@cdo/apps/componentLibrary/dropdown/_CustomDropdown';
+import FontAwesomeV6Icon, {
+  FontAwesomeV6IconProps,
+} from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
+
+import moduleStyles from '@cdo/apps/componentLibrary/dropdown/customDropdown.module.scss';
+
+export interface ActionDropdownOption {
```

Comment:
Looks like these props are kind of similar to `CheckboxOption` and `IconDropdownOption` - would it be helpful to extract the common props into some base interface?

Reply:
Yeah, there're some similarities and also some differences between these three. Will take a look into it in a follow up Dropdowns update